build.sh

```diff
@@ -401,6 +410,15 @@ build_one() {
                 INCLUDE_PATH="${root}/include" \
                 LIBRARY_PATH="${root}/lib" install
         ;;
+    libzip)
+	do_cmake \
+                -DBUILD_PKGCONFIG_FILES=ON \
+                -DBUILD_DOC=ON \
+                -DZLIB_DLL=ON \
+                -D_WIN32=ON
```

Comment:
Ok. Now I remember. It was a while ago, and this was the reason why I was using the double approach to "somehow" produce a libzip.pc - because the CMake script doesn't seem to do that. Still looking for an elegant solution. Giving up on requirement checks would be a bad idea, would it?